src/components/MailboxPicker.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<Treeselect
+		ref="Treeselect"
+		:options="mailboxes"
+		:multiple="false"
+		:clearable="false"
+		v-bind="$attrs"
+		v-on="$listeners" />
+</template>
+<script>
+import Treeselect from '@riophae/vue-treeselect'
+import '@riophae/vue-treeselect/dist/vue-treeselect.css'
+
+export default {
+	name: 'MailboxPicker',
+	components: {
+		Treeselect,
+	},
+	props: {
+		accountid: {
```

Comment:
```suggestion
		accountId: {
```

or pass the full `account` object. We often pass the full object if it's always known in the parent component.

Reply:
The full account might be the best solution. Done.